addons/escoria-core/game/core-scripts/esc/esc_room_manager.gd

```diff
@@ -32,8 +32,146 @@ func register_reserved_globals() -> void:
 			RESERVED_GLOBALS[key])
 
 
+# Performs the actions needed in order to change the current scene to the one 
+# specified by room_path.
+#
+# #### Parameters
+#
+# * room_path: Node path to the room that is to become the new current room.
```

Comment:
Please use dashes instead of asterisks. I think, we're using dashes everywhere else.